mobile/bindings.go

```diff
@@ -0,0 +1,58 @@
+// +build ios android
+
+package lndmobile
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	flags "github.com/jessevdk/go-flags"
+	"github.com/lightningnetwork/lnd"
+)
+
+// Start starts lnd in a new goroutine.
+//
```

Comment:
Assumption is that it will die with the parent process (it is actually the same process IIRC). It can also be stopped by calling the `Stop` RPC.

Haven't investigated if we can rely on the signals on mobile platforms. Can be investigated along with tower support.